### PR TITLE
Make tinymce functionality equivalent to ckeditor out of the box

### LINKED ISSUE #15887: [Feature]: Add Search and Replace plugin to TinyMCE
### Feature description

As it was also available in the CKEditor, I would like to have the search and replace plugin activated by default (or some documentation on how to activate it) 

https://www.tiny.cloud/docs/plugins/opensource/searchreplace/

We depend heavily on this functionality, I can open a PR myself but I need some guidance on how to active the plugin.

Thanks.

### PR DESCRIPTION
**Pull request by Daniel-Ateles**
Added some basic plugins

### CODE REVIEW

**Comment by rubanooo** [comment]
Closes https://github.com/pimcore/pimcore/issues/15887 when merged 👍

**Comment by rubanooo** [comment]
@wisconaut Sorry for being this guy, but this has been open for months and seems like really a easy merge. We would really like this functionality. Anything I can do to get this merged?

**Comment by ghost** [comment]
@rubanooo Thanks for reminding and no worries. It's already on our list for one of the next time slots, so hopefully done soon.

**Comment by robertSt7** [comment]
Hi @rubanooo when I try to build the js files. I get the following error: 

`
Module build failed: Module not found:
"./public/assets/tinymce.js" contains a reference to the file "tinymce/plugins/hr".
This file can not be found, please check it for typos or update it if the file got moved.
` 

Could you please take a look at that? Thanks

File: bundles/TinymceBundle/public/assets/tinymce.js [hunk 1]
```diff
@@ -16,4 +16,10 @@ import 'tinymce/plugins/insertdatetime';
...
 import 'tinymce/plugins/help';
 import 'tinymce/plugins/lists';
-import 'tinymce/plugins/code';
\ No newline at end of file
+import 'tinymce/plugins/code';
+
+import 'tinymce/plugins/searchreplace';
+import 'tinymce/plugins/advlist';
+import 'tinymce/plugins/charmap';
+import 'tinymce/plugins/hr';
```
Comment (blankse) [inline bundles/TinymceBundle/public/assets/tinymce.js]: This plugin is missing
```suggestion
```

Comment (rubanooo) [reply bundles/TinymceBundle/public/assets/tinymce.js]: @Daniel-Ateles Could you apply this change?

**Comment by robertSt7** [comment]
@Daniel-Ateles Could you please also add the plugins to the plugin list here? Thanks 
https://github.com/pimcore/pimcore/blob/28c211230f8c649575a035aaae83bd57c6f232f3/bundles/TinymceBundle/public/js/editor.js#L90-L93

**Comment by blankse** [comment]
@robertSt7 I created an alternative PR https://github.com/pimcore/pimcore/pull/17319

**Comment by Daniel-Ateles** [comment]
@robertSt7 done!

File: bundles/TinymceBundle/public/assets/tinymce.js (hunk 1 above)
Comment (Daniel-Ateles) [reply bundles/TinymceBundle/public/assets/tinymce.js]: > @Daniel-Ateles Could you apply this change?

Done!

**Comment by robertSt7** [comment]
@Daniel-Ateles Thank you anyway for coming up with this and doing the first steps
closed in favor of https://github.com/pimcore/pimcore/pull/17319

**Pull request closed without merging**